docs/getting_started/prepare_environment.md

```diff
@@ -1,61 +1,39 @@
 # Tutorial: Prepare your Environment
 
-Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine using Ubuntu 20.04.
+Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine using Ubuntu 22.04 and Azure Linux 2.0.
 
 - [Install Tools](#install-tools)
-- [Clone CBL-Mariner](#clone-cbl-mariner-and-build-the-toolkit)
-- [Clone CBL-MarinerTutorials](#clone-cbl-marinertutorials-repo-and-extract-the-toolkit)
+- [Clone AzureLinux](#clone-azure-linux-and-build-the-toolkit)
+- [Clone AzureLinux-Tutorials](#clone-azure-linux-tutorials-repo-and-extract-the-toolkit)
 
 ## Install Tools
 
-These tools are required for building both the toolkit and the images built from the toolkit.  These are the same [prerequisites needed for building CBL-Mariner](https://github.com/microsoft/CBL-Mariner/blob/-/toolkit/docs/building/prerequisites.md).
+Get the prerequisites for building the tools. Instructions are provided for both Ubuntu and Azure Linux at [prerequisites needed for building Azure Linux](https://github.com/microsoft/azurelinux/blob/-/toolkit/docs/building/prerequisites.md).
 
-```bash
-# Add a backports repo in order to install the necessary version of Go.
-sudo add-apt-repository ppa:longsleep/golang-backports
-sudo apt-get update
-
-# Install required dependencies.
-sudo apt -y install git make tar wget curl rpm qemu-utils golang-1.20-go genisoimage python bison gawk
-
-# Recommended but not required: `pigz` for faster compression operations.
-sudo apt -y install pigz
+## Clone Azure Linux and Build the Toolkit
 
-# Fix go 1.17 link
-sudo ln -vsf /usr/lib/go-1.17/bin/go /usr/bin/go
-
-# Install Docker.
-curl -fsSL https://get.docker.com -o get-docker.sh
-sudo sh get-docker.sh
-sudo usermod -aG docker $USER
-```
+To build the `Azurelinux-Tutorials` repository you will need the same toolkit and makefile from the `Azurelinux` repository.  So, first clone `Azurelinux`, and then checkout the stable release of interest (e.g. 3.0-stable), then build the toolkit.
 
-**You will need to log out and lock back in** for user changes to take effect.
-
-## Clone CBL-Mariner and Build the Toolkit
-
-To build the CBL-MarinerTutorials repository you will need the same toolkit and makefile from the CBL-Mariner repository.  So, first clone CBL-Mariner, and then checkout the stable release of interest (e.g. 2.0-stable), then build the toolkit.
-
-### Example for CBL-Mariner 2.0 Toolkit
+### Example for Azurelinux 3.0 Toolkit
 
 ```bash
-git clone https://github.com/microsoft/CBL-Mariner.git
-pushd CBL-Mariner/toolkit
-git checkout 2.0-stable
-sudo make package-toolkit REBUILD_TOOLS=y
+git clone https://github.com/microsoft/azurelinux.git
+pushd azurelinux/toolkit
+git checkout 3.0-stable
+sudo make -j20 package-toolkit REBUILD_TOOLS=y
 popd
```

Comment:
Minor suggestion: maybe we can make the example a bit shorter to focus on the "meat" of it better.
```suggestion
git clone -b 3.0-stable https://github.com/microsoft/azurelinux.git
sudo make -C azurelinux/toolkit -j20 package-toolkit REBUILD_TOOLS=y
```